lib/api_repo.js

```diff
@@ -77,7 +77,7 @@ function ApiRepo(opts) {
   this.pkgPrefix = opts.pkgPrefix;
   this.overridePlugins = opts.overridePlugins;
   this.zipUrl = opts.zipUrl;
-  this.isGoogleApi = !!opts.isGoogleApi;
+  this.isGoogleApi = !!opts.buildCommonProtos;
```

Comment:
Hm... I see. I reviewed the code base and it looks like better to keep this as `isGoogleApi` for now.

Two notes:
- it seems `zipUrl` will also be not used anymore actually -- mostly the parameters are supplied properly from artman. Are you thinking to remove it?
- probably we could eventually refactor the code so that buildCommonProtos won't require isGoogleApi.